test/units/executor/test_task_executor.py

```diff
@@ -323,5 +323,5 @@ def _get(*args, **kwargs):
         with patch.object(action_loader, 'get', _get):
             mock_templar = MagicMock()
             res = te._poll_async_result(result=dict(ansible_job_id=1), templar=mock_templar)
-            self.assertEqual(res, dict(finished=1))
+            self.assertTrue(('finished', 1) in res.items())
```

Comment:
the mock task has no_log set to true for some reason, so the output dict with the propagated no_log value no longer equals the static one. changed to just ensure that finished is in the results instead.